CONTRIBUTING.md

```diff
@@ -76,7 +76,16 @@ For small changes to a single file, you can edit directly in GitHub by clicking
 **Edit this file** button. After forking the repository, follow the instructions
 in [Editing files][].
 
-For everything else, follow the
+However, formatting may still be needed, like reducing line lengths in the
+edited file. The options for fixing formatting are:
+
+- Checking out the project and running the CLI scripts mentioned in
+  [Submitting a change](#submitting-a-change).
+- Commenting `/fix:format` on your pull request to trigger an automated script.
+  This requires a unique branch name, which can be edited under _View all
+  branches_ in your fork.
+
```

Comment:
Let's add this in for now, but eventually I want to fix the automation script: we know what the problem is (if the branch name is used upstream as well the script gets "confused") and what the solution is (set the remote for the commit accordingly to the fork)